pdal/SpatialReference.cpp

```diff
@@ -475,40 +486,32 @@ std::string SpatialReference::prettyWkt(const std::string& wkt)
     if (!srs)
         return outWkt;
 
-    char *buf = nullptr;
-    srs->exportToPrettyWkt(&buf, FALSE);
-
-    outWkt = buf;
-    CPLFree(buf);
+    const char* apszOptions[] = { "MULTILINE=YES", nullptr }; // equivalent to exportToPrettyWkt
+    outWkt = exportToWkt(srs.get(), apszOptions);
     return outWkt;
 }
 
-std::string SpatialReference::getWKT1() const
+std::string SpatialReference::getWKT1(bool throwIfEmpty) const
```

Comment:
what precipitated this change?

Reply:
TLDR; "I tried to keep the same behavior when wkt2 is not enabled"

In the function `LasWriter::addWktVlr()`, `getWKT1()` was throwing an exception if the result of `exportToWkt` is empty. That stopped the whole process. In case the wkt2 is stored, then we do not want to throw. In that case it just not write the wkt1 and geotiff.

I do not like this solution so much. I can either:
- do a try-catch around, removing the parameter `throwIfEmpty`
- change the behaviour of `getWKT1()` and do not throw.

See that if no srs is provided, there is no exception. The exception only is thrown if the existing srs does not produce an output in `exportToWkt` with `FORMAT=WKT1_GDAL`